### PR TITLE
JENA-2159: Add Schema.org vocabulary

### PR DESCRIPTION
**Pull request by ebremer**

### CODE REVIEW

File: jena-core/src/main/java/org/apache/jena/vocabulary/SO.java [hunk 1]
```diff
@@ -0,0 +1,2366 @@
...
+
+/**
+ *  Schema.org is a collaborative, community activity with a mission to create, maintain, and promote schemas for structured data on the Internet, on web pages, in email messages, and beyond.
+ *  <p>
+ *	See <a href="https://schema.org/">Schema.org</a>.
+ *  <p>
+ *  <a href="https://schema.org/">Base URI and namepace</a>.
+ */
+    private static final Model m = ModelFactory.createDefaultModel();
+    public static final String NS = "https://schema.org/";
```
Comment (afs) [inline jena-core/src/main/java/org/apache/jena/vocabulary/SO.java:36]: FYI:
schema.org might be "http://" as well. There is no right answer here - it's at the whim of the JSON-LD context. So I asked and got a response that "https" is slightly better. We'll see.

**Review by afs — COMMENTED** [review]
The file name "SO" is a bit terse :-)

SchemaDO.java
SchemaDotOrg.java

I would expect that it would be static import much of the time so the name length isn't so crticial. Maybe.

File: jena-core/src/main/java/org/apache/jena/vocabulary/SO.java (hunk 1 above)
Comment (ebremer) [reply jena-core/src/main/java/org/apache/jena/vocabulary/SO.java]: I changed the name.

**Review by afs — APPROVED** [review]
Recorded as [JENA-2159.](https://issues.apache.org/jira/browse/JENA-2159).

**Pull request merged**